web/app/static/js/main.js

```diff
@@ -6,7 +6,10 @@ document.querySelectorAll(".single-search-form").forEach((form) =>
 
     await fetch("/search_column", {
       method: "POST",
-      body: JSON.stringify({ col_name: e.target.dataset.search_value }),
+      body: JSON.stringify({
+        col_name: e.target.dataset.search_value,
+        current_href: window.location.href,
```

Comment:
I'd suggest we extract model name from href here, rather then passing  whole href to backend

Reply:
It would just transfer potential error from backend to frontend. I decided to handle it on backend